src/koans/13_creating_functions.clj

```diff
@@ -5,31 +5,34 @@
 
 (meditations
   "One may know what they seek by knowing what they do not seek"
-  (= [__ __ __] (let [not-a-symbol? (complement symbol?)]
+  (= [true false true] (let [not-a-symbol? (complement symbol?)]
                   (map not-a-symbol? [:a 'b "c"])))
 
   "Praise and 'complement' may help you separate the wheat from the chaff"
   (= [:wheat "wheat" 'wheat]
-       (let [not-nil? ___]
+       (let [not-nil? (fn [x] x)]
```

Comment:
More idiomatic and elegant way is to use [complement](https://clojuredocs.org/clojure.core/complement) function. For example `(complement nil?)`.

Reply:
She's beautiful capn.